docs/release/updating-to-osg-25.md

```diff
@@ -49,6 +55,92 @@ Updating the OSG Repositories
 
 1.  Continue on to any update instructions that match the role(s) that the host performs.
 
+Updating Your OSG Access Point
+------------------------------
+
+In OSG 25, some manual configuration changes may be required for an OSG Access Point (APs).
+
+#### HTCondor ####
+
+Consult the [HTCondor upgrade section](#updating-your-htcondor-hosts) for details on updating your HTCondor configuration.
+
+### Restarting HTCondor ###
+
+After updating your RPMs, restart your HTCondor service:
+
+```console
+root@host # systemctl restart condor
+```
+
```

Comment:
Since there's nothing OSPool AP specific, I think we can just strike this section and just call out OSPool APs in the "HTCondor hosts" section

Reply:
Done